internal/resource/config/authenticationpolicycontract/authentication_policy_contract_resource.go

```diff
@@ -23,6 +25,19 @@ var (
 	_ resource.Resource                = &authenticationPolicyContractResource{}
 	_ resource.ResourceWithConfigure   = &authenticationPolicyContractResource{}
 	_ resource.ResourceWithImportState = &authenticationPolicyContractResource{}
+
+	objAttrType = map[string]attr.Type{
```

Comment:
nit: you could name these variables to be clearer for what they are for in the schema

Reply:
pushed